### DISCUSSION TITLE
No redirect for links

### DISCUSSION THREAD

**Discussion by lucasfth** [s1lvax/route#35] [Q&A]
Hi. This seems like an awesome project.
The only problem is that right now I seem to be unable to be redirected by clicking a link. So on the below image when clicking on portfolio nothing happens.
<img width="1800" alt="image" src="https://github.com/user-attachments/assets/b4a9ab70-2642-4a8f-94c5-a6d9a9727c32">
I also tried on this [page](https://route2.dev/s1lvax) and it did not work either as well as also tested on another browser (have now tried on Arc as well as Safari).

Is this a general issue or something that is only an issue for me?

**Comment by lucasfth** [comment]
Seems to work now. Not sure if the problem was on my side. Thus closing

**Reply by s1lvax** [reply]
It was on my end, I changed the design and forgot the anchor tag :).
